### ISSUE TITLE
Ignore SC2016 for "busybox awk" just like it is ignored for simple "awk" call

### ISSUE THREAD

**Issue by S-trace** [koalaman/shellcheck#819]
#### For bugs
- Rule Id : SC2016
- My shellcheck version: online
- [+] I tried on shellcheck.net and verified that this is still a problem on the latest commit
#### Here's a snippet or screenshot that shows the problem:

```sh
awk '{print $1}'
busybox awk '{print $1}'
```

#### Here's what shellcheck currently says:
$ shellcheck myscript
 
Line 2:
busybox awk '{print $1}'
            ^-- SC2016: Expressions don't expand in single quotes, use double quotes for that.

$ 

#### Here's what I wanted or expected to see:
$ shellcheck myscript
No issues detected!

$

**Comment by koalaman** [comment]
Good point. Fixed in 35c74e4
